Jenkinsfile

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'govuk_navigation_helpers'
```

Comment:
Unused constant? I think the other repos just use this in the Integration deployment step, which this gem doesn't have.